x-pack/test/apm_api_integration/tests/alerts/anomaly_alert.spec.ts

```diff
@@ -23,23 +24,24 @@ export default function ApiTest({ getService }: FtrProviderContext) {
   const logger = getService('log');
 
   const synthtraceEsClient = getService('synthtraceEsClient');
-  // FLAKY https://github.com/elastic/kibana/issues/160298
-  registry.when.skip(
+  registry.when(
     'fetching service anomalies with a trial license',
     { config: 'trial', archives: [] },
     () => {
-      const start = '2021-01-01T00:00:00.000Z';
-      const end = '2021-01-08T00:15:00.000Z';
+      const start = moment().subtract(1, 'days').toISOString();
+      const end = moment().toISOString();
```

Comment:
Shouldn't this also be timestamps? Then you don't have to convert them later

Reply:
Good point